package.json

```diff
@@ -89,6 +85,7 @@
     "webpack-cli": "^4.0.0"
   },
   "dependencies": {
+    "@types/node-fetch": "^2.5.7",
```

Comment:
@bcoe I don't usually like doing this, but with exposing a `fetch` TypeScript type, it was required to compile.  `node-fetch` is bundling their own types starting in v3, which is currently in beta, then this can go away.